src/main/java/uk/gov/ida/matchingserviceadapter/rest/matchingservice/EidasMatchingDataset.java

```diff
@@ -0,0 +1,111 @@
+package uk.gov.ida.matchingserviceadapter.rest.matchingservice;
+
+import org.joda.time.LocalDate;
+
+public class EidasMatchingDataset {
+    private final Address address;
+    private final LocalDate dateOfBirth;
+    private final String firstName;
+    private final String gender;
+    private final String familyName;
+    private final String birthName;
+    private final String placeOfBirth;
+
+    public EidasMatchingDataset(Address address, LocalDate dateOfBirth, String firstName, String familyName, String birthName, String gender, String placeOfBirth) {
+        this.address = address;
+        this.dateOfBirth = dateOfBirth;
+        this.firstName = firstName;
+        this.gender = gender;
+        this.familyName = familyName;
+        this.birthName = birthName;
+        this.placeOfBirth = placeOfBirth;
+    }
+
+    public Address getAddress() {
+        return address;
+    }
+
+    public LocalDate getDateOfBirth() {
+        return dateOfBirth;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getGender() {
+        return gender;
+    }
+
+    public String getFamilyName() {
+        return familyName;
+    }
+
+    public String getBirthName() {
+        return birthName;
+    }
+
+    public String getPlaceOfBirth() {
+        return placeOfBirth;
+    }
+
+    public static class Address {
```

Comment:
Two things:
- This is also a DTO
- The standard Verify AddressDto is a class on its own, but the EIDAS one isn't. It's not quite consistent...